src/main/java/page_objects/RegisterPage.java

```diff
@@ -0,0 +1,30 @@
+package page_objects;
+
+import common.element.Button;
+import common.element.Label;
+import common.element.TextBox;
+import model.User;
+import org.openqa.selenium.By;
+
+public class RegisterPage extends GeneralPage{
+  private final TextBox txtEmail = new TextBox(By.id("email"));
+  private final TextBox txtPassword = new TextBox(By.id("password"));
+  private final TextBox txtConfirmPassword = new TextBox(By.id("confirmPassword"));
+  private final TextBox txtPid = new TextBox(By.id("pid"));
+
+  private final Button btnRegister = new Button(By.cssSelector(".form-actions input"));
```

Comment:
this xpath is pretty weak, use something more specific and descriptive

Reply:
I fixed it. Thank you.